pandas/_libs/tslibs/_timestamp.pyx

```diff
@@ -0,0 +1,418 @@
+# -*- coding: utf-8 -*-
+"""
+_Timestamp is in its own module to prevent the circular imports
```

Comment:
can you revise this entire doc-string. This is a c-defined subclass of datetime.datetime, not user visible.

Reply:
I've revised the doc-string.  Not sure if it is exactly what you were looking for.